test/database.ts

```diff
@@ -40,7 +40,13 @@ import {protos} from '../src';
 import * as inst from '../src/instance';
 import RequestOptions = google.spanner.v1.RequestOptions;
 import EncryptionType = google.spanner.admin.database.v1.RestoreDatabaseEncryptionConfig.EncryptionType;
-import {BatchWriteOptions} from '../src/transaction';
+import {
```

Comment:
Can we add a test (potentially to the `spanner.ts` file using the mock server) that verifies that:
1. Using `writeAtLeastOnce` does not trigger a `BeginTransaction` RPC.
2. Using `writeAtLeastOnce` triggers a `Commit` RPC that uses a single-use read/write transaction.